Makefile

```diff
@@ -148,7 +149,8 @@ frontend-tests: node_modules/.uptodate
 
 DOCKER_TAG = dev
 
-build/manifest.json: node_modules/.uptodate
+.PHONY: build
+build: node_modules/.uptodate
```

Comment:
There are other references to this target in this file which will need to be updated. I also suggest naming this `frontend-build` for consistency with the other `frontend-*` targets and also to avoid confusion in this project that has both a frontend and backend component and where the main output is a Docker image.